templates/github/.ci/scripts/tweet.py.j2

```diff
@@ -0,0 +1,16 @@
+import os
+import sys
+from tweepy import Client
+
+release_version = sys.argv[1]
+if release_version.endswith(".0"):
+    client = Client(
+        consumer_key=os.getenv("TWITTER_API_KEY"),
+        consumer_secret=os.getenv("TWITTER_API_KEY_SECRET"),
+        access_token=os.getenv("TWITTER_ACCESS_TOKEN"),
+        access_token_secret=os.getenv("TWITTER_ACCESS_TOKEN_SECRET"),
+    )
+    link = "https://docs.pulpproject.org/{{ plugin_snake }}/changes.html"
```

Comment:
did you mean plugin_name?

Reply:
nope, plugin_snake for ensuring the _
https://docs.pulpproject.org/pulp_2to3_migration/